src/pack-n-test.ts

```diff
@@ -112,14 +130,12 @@ export async function packNTest(options: TestOptions) {
     );
 
     // Poupulate test code.
-    const filename = isJSCodeSample(sample) ? 'index.js' : 'index.ts';
-    const code = isJSCodeSample(sample) ? sample.js : sample.ts;
+    const {code, filename} = getSample(sample);
     await writeFileP(path.join(installDir, filename), code, 'utf-8');
 
-    // If TypeScript, compile it.
-    if (!isJSCodeSample(sample)) {
+    if (sample.ts) {
       // TODO: maybe make it flexible for users to pass in typescript config.
-      await execa('node_modules/.bin/tsc', ['--strict', 'index.ts'], {
```

Comment:
Relying on `node_modules` paths is not a great idea (and not a supported usage in general), `npx` is going to be a better (and supported) alternative to achieve this.

Reply:
As long as `pack-n-play` is CJS you could probably do `path.join(require.resolve('typescript'), '..', '..', 'bin', 'tsc')` but I don't know if it's actually better :)

Reply:
I agree that `npx` is more straightforward.